init.lua

```diff
@@ -137,13 +139,14 @@ local function entry(_, job)
 	if job.args[1] == "auto" or job.args[1] == "history" then
 		shell_value = shell_env:lower()
 	elseif job.args[1] == "custom" then
-		if job.args[2] == "--wait" or job.args[2] == "-w" then
-			shell_value = job.args[3]
-			cmd = job.args[4]
+		if job.args.wait then
```

Comment:
I guess this `if` statement is no longer needed

Reply:
oh yeah former to change this.